content/W2016/W16-event-stephanie-talk.md

```diff
@@ -0,0 +1,36 @@
+Title: Circuit Breaker: How my Love for Writing Brought me to Tech
```

Comment:
Circuit Breaker? I don't know what that means

Also, My and Me should be capitalized